pkgs/applications/accessibility/svkbd/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchurl
+, pkg-config
+, xlibs
+, layout ? null
+, patches ? [ ]
```

Other suckless packages use
```suggestion
, patches ? null
```
Does that make a difference?

---

``null != [ ]`` but setting it to [] gives people a type hint which is preferable. null could avoid rebuilds which is irrelevant here.